rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -24,6 +122,17 @@ def __init__(
         batch_length_T: int = 1,
         alpha: float = 1.0,
     ):
+        """Initializes a `PrioritizedEpisodeReplayBuffer` object
+
+        Args:
+            capacity: The total number of timesteps to be storable in this buffer.
+                Will start ejecting old episodes once this limit is reached.
+            batch_size_B: The number of rows in a SampleBatch returned from `sample()`.
+            batch_length_T: The length of each row in a SampleBatch returned from
```

Comment:
Can we specify here, how this arg is related to n-step? My understanding is: Not at all, correct?

So, batch_length_T is the same as in DreamerV3, meaning if >1, then my returned batch will contain B rows, each row consisting of T consecutive-in-an-episode(!) n-step tuples, correct?

If so, can we give these details here in the description of the arg?

Reply:
Ar rhis moment `batch_öength_T=1` is the only possible option. As this is a method inherited from `EpisodeReplayBuffer` this argument has to be there.